dashboard/db/migrate/20160430111030_add_kind_to_survey_results.rb

```diff
@@ -0,0 +1,13 @@
+class AddKindToSurveyResults < ActiveRecord::Migration
+  def change
+    add_column :survey_results, :kind, :string, :after => :user_id
+    add_index :survey_results, :kind
+
+    reversible do |direction|
+      direction.up do
+        # On 2016-04-30, existing rows are the diversity survey from March 2016.
+        SurveyResult.update_all(kind: 'Diversity2016')
```

Comment:
I thought maybe if you used reversible then you needed to provide up & down explicitly, whereas change is the one that can automatically go in both directions.  At any rate, this can easily be confirmed locally first, which I highly recommend.